apollo-compiler/src/test/graphql/com/example/custom_scalar_type/CustomScalarTypeExpected.java

```diff
@@ -0,0 +1,20 @@
+package com.example.custom_scalar_type.type;
+
+import com.apollographql.android.api.graphql.ScalarTypeMapping;
+import java.lang.Class;
+import java.lang.String;
+import java.util.Date;
+import javax.annotation.Generated;
+
+@Generated("Apollo GraphQL")
+public enum CustomScalarType implements ScalarTypeMapping {
+  DATE {
+    public String type() {
```

Comment:
this `type` is type defined in GraphQl schema: like `Date`, `DateTime`, `SomeOtherType`.
`clazz` - is actually java mapping class for this `type`